extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -164,25 +187,24 @@ export class AmpSlideScroll extends BaseSlides {
 
   /** @override */
   onLayoutMeasure() {
-    /** @private {number} */
     this.slideWidth_ = this.getLayoutWidth();
 
-    /** @private {number} */
     this.previousScrollLeft_ = this.slidesContainer_./*OK*/scrollLeft;
   }
 
   /** @override */
   layoutCallback() {
-    if (this.slideIndex_ == null) {
+    if (this.slideIndex_ === null) {
```

Comment:
ok i found the real issue, had to look at the binary with pseudo_names, still need to figure out why its doing it.

the optimization its doing is really weird